br/tests/br_debug_meta/run.sh

```
@@ -32,36 +32,49 @@ run_sql "$table_region_sql"
 
 row_count_ori=$(run_sql "SELECT COUNT(*) FROM $DB.$TABLE;" | awk '/COUNT/{print $2}')
 
-# backup table
+# backup table with backupmetav2
 echo "backup start..."
 run_br --pd $PD_ADDR backup table --db $DB --table $TABLE -s "local://$TEST_DIR/$DB"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)